carbonmark-data/components/pages/DetailPage/index.tsx

```diff
@@ -0,0 +1,22 @@
+import OverviewCard from "components/OverviewCard";
+import Skeleton from "components/Skeleton";
+import { Suspense } from "react";
+/**
+ * A UI layout component to position detail pages content
+ */
+export default function DetailPage(props: {
+  chart: React.ReactNode | Promise<React.ReactNode>;
```

Comment:
Does this need to be a required param or could it be marked as optional? The reason I ask, I see in a few different places empty tags `<></>` are been passed in for the chart param?

Reply:
This should be mandatory. The empty tags are placeholders I put there until we develop the components that will be used.